packages/beacon-ui/src/components/bug-report-form/index.tsx

```diff
@@ -119,32 +197,24 @@ const BugReportForm = (props: any) => {
       title: title(),
       sdkVersion: SDK_VERSION,
       description: description(),
-      steps: steps(),
+      steps: '<#EMPTY#>',
       os: currentOS(),
       browser: currentBrowser(),
-      localStorage: JSON.stringify(beaconState),
-      wcStorage: JSON.stringify(wcState)
+      localStorage: JSON.stringify(clean(beaconState)),
+      wcStorage: JSON.stringify(clean(wcState))
     }
 
-    const options = {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json'
-      },
-      body: JSON.stringify(request)
-    }
-
-    fetch('https://beacon-backend.prod.gke.papers.tech/bug-report/save', options)
+    sendRequest('https://beacon-backend.prod.gke.papers.tech/bug-report/save', 'POST', request)
```

Comment:
I think we have the base URL in the config/constants somewhere, we should use that (or put it there if it's not).